interface.go

```diff
@@ -266,6 +266,11 @@ type Config struct {
 	// MaxConnectionReceiveWindow is the connection-level flow control window for receiving data.
 	// If this value is zero, it will default to 15 MB.
 	MaxConnectionReceiveWindow uint64
+	// AllowConnectionWindowIncrease is called every time the connection flow controller attempts
+	// to increase the connection flow control window.
+	// If set, the caller can prevent an increase of the window. Typically, it would do so to
+	// limit the memory usage.
+	AllowConnectionWindowIncrease func(sess Session, delta int) bool
```

Comment:
When calling this, we hold at least one mutex (the one in the connection flow controller), and are within the session run loop. Is it safe to call all Session functions from here, or are there any restrictions we need to document?

Reply:
Yeah, you should call any functions on the Session, are any Stream. I added a comment.